deployments/Dockerfiles/autonomy/requirements.txt

```diff
@@ -0,0 +1,9 @@
+watchdog==2.1.6
```

Comment:
I remove the usage of `pipenv` since we were using the `--system` flag anyway, so this was I converted the Pipfile to requirement.txt

Reply:
Is there a test to ensure that this aligns with th pipefile for the root dir?

Reply:
Ok, that explains my comment above

Reply:
> Is there a test to ensure that this aligns with th pipefile for the root dir?

I'll open an issue for this once the PR is merged